src/status_im/subs/activity_center.cljs

```diff
@@ -6,35 +6,36 @@
             [clojure.string :as string]))
 
 (re-frame/reg-sub
- :activity-center/notifications-read
- (fn [db]
-   (get-in db [:activity-center :notifications-read :data])))
+ :activity-center/notifications
```

Comment:
just wondering how its different from `:activity.center/notifications` why do we have both

Reply:
I'm not sure I understand the question correctly, but the idea is that all new activities are managed under a different root state called `:activity-center` (with a dash, not with a dot). I know it's hard to spot this, but eventually the current (old) notification center will be removed and with it, the subscription `:activity.center/notifications` (with dot).

This was a strategy to code in the new AC without ever messing around with the existing app state for the old Notification Center.

Does that answer your question?

Reply:
yes, thank you